responsive.css

```diff
@@ -0,0 +1,367 @@
+@media only screen and (min-width: 1025px) {
+  .nav2 {
+    background-color: #fff;
+    color: #aaa;
+    margin: 0 auto;
+    display: flex;
+    justify-content: space-between;
+    font-weight: 500;
+    font-size: 15px;
+  }
+
+  .nav1 {
+    background-color: #131313;
+    color: #9f9f9f;
+    margin: 0 auto;
+    display: flex;
+    justify-content: space-between;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.flex{
  display:flex;
}
```
**Html file**

```
<div class="flex">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_